examples/literate/D_simulatingFloatingPlatforms.jl

```diff
@@ -0,0 +1,394 @@
+# # [Simulating Floating Platforms](@id simple2)
+# 
+# In this example, we make the OWENS model more complex by simulation a floating wind
+# turbine instead of a land-based one. This involves coupling two meshes together: one
+# representing the wind turbine (this will be similar to meshes used in previous tutorials),
+# and one representing the floating platform.
+#
+# The wind turbine mesh (or "topside") will be created similarly to the previous tutorial,
+# though we will overwrite some of the inputs so the simulation works better for a floating system.
+# This will help with revealing some more internals of OWENS and the potential inputs
+# available to users, which is helpful for explaining how the floating platform mesh
+# (or "bottom side") is defined and differs from the topside.
+
+import PyPlot
+
+import OWENS
+import OWENSFEA
+
+path = runpath = splitdir(@__FILE__)[1]
```

Comment:
You'll need to change this to runpath = path = "/home/runner/work/OWENS.jl/OWENS.jl/examples/literate" so it works within the ci/cd environment.  I wasn't ever able to figure out how to get it to work with a relative path.